frontend/views/containers/dashboard/GroupMembersAllModal.vue

```diff
@@ -89,17 +96,30 @@ export default {
   },
   computed: {
     ...mapGetters([
+      'currentGroupState',
       'groupProfiles',
       'globalProfile',
-      'groupMembersCount'
+      'groupMembersCount',
+      'groupMembersPending',
+      'ourUsername'
     ]),
+    weJoinedMs () {
+      return new Date(this.currentGroupState.profiles[this.ourUsername].joinedDate).getTime()
+    },
+    allMembers () {
+      return Object.keys({ ...this.groupMembersPending, ...this.groupProfiles })
```

Comment:
If you just want the keys of these two objects, then there's a faster way of doing this:

```js
return Object.keys(this.groupMembersPending).concat(Object.keys(this.groupProfiles))
```

Reply:
Hum... that's what I'm doing here. 😅 The difference to your suggestion is that I'm using destructuring (`...`) to create the array instead of using `Object.keys` twice and concat.

Reply:
> And using destructuring is faster: https://jsperf.com/object-keys-concact-vs-destructuring/1

Huh. That's surprising. OK we'll keep it your way 😄  👍